src/js/profile/mobile/widget/DropdownMenu.js

```diff
@@ -1000,7 +1000,7 @@
 					elOptionContainer.addEventListener("focusin", self._focusBound); // bubble
 					elOptionContainer.addEventListener("focusout", self._blurBound); // bubble
 					if (ui.screenFilter) {
-						ui.screenFilter.addEventListener("vclick", self._toggleMenuBound);
```

Comment:
was there any particular reason to change vclick to vmousedown?

Reply:
Yes, this is intended. The drop down menu has to close immediately after overlay touch. I verified this behaviour in Android app with AppBar (eg. Messages).